main.py

```diff
@@ -32,6 +32,8 @@ def list_entity(entity):
     return call(f"{BASE_URL}{entity}/").get("results")
 
 
+st.header("API do Star Wars!!")
```

Comment:
Eu acho que a gente pode tirar os pontos de exclamação

```suggestion
st.header("API do Star Wars")
```